src/hardware/ipxserver.cpp

```diff
@@ -212,10 +223,39 @@ bool IPX_StartServer(uint16_t portnum)
 		ipxServerSocket = SDLNet_UDP_Open(portnum);
 		if(!ipxServerSocket) return false;
 
-		for (uint16_t i = 0; i < SOCKETTABLESIZE; ++i)
-			connBuffer[i].connected = false;
+		for (auto& i : connBuffer) {
+			i.connected = false;
+		}
+
+		if (!socket_set) {
+			socket_set = SDLNet_AllocSocketSet(1);
+			if (!socket_set) {
+				LOG_ERR("IPXServer: %s", SDLNet_GetError());
+				return false;
+			}
+			if (SDLNet_UDP_AddSocket(socket_set, ipxServerSocket) == -1) {
+				LOG_ERR("IPXServer: %s", SDLNet_GetError());
```

Comment:
@kklobe minor: We do ALL_UPPER_CASE log prefixes, so `IPXSERVER` would be more consistent.

Reply:
ah, shikes, I don't know how that got in there. Worth a quick add-on PR?

Reply:
Up to you; if you feel like it, sure why not 😄 Or the next person touching this file can do a cleanup sweep.

I was mainly commenting on these minor issues as an "information dissemination" exercise. 99% of the work here was making it threaded and that's great—I'm just reflecting on the minor superficial stuff 😄 

Some other common things kcgen and I use to correct on sight:

```cpp
all_vars_lower_snake_case (basically any var definition anywhere)
enum class MyEnum { SomeEnumValue, AndAnotherOne }
static void i_am_a_private_function
struct UpperCamel { ... } (no weird _t postfix)
```

Reply:
ok, I'm going to create a `.clang-tidy` that flags these